charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.2/src/templates/dotnet-imagestream.yaml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
@tmds you're currently failing helm lint because of this line in both templates. Any reason why this needs to be here? Helm lint is the only thing complaining - helm template and install likely do the right thing.

Reply:
This isn't functionally needed. I've removed it.